### PR TITLE
i18n(es): Updated `from-gatsby.mdx`,`from-nuxtjsmdx`, `from-nextjs.mdx`, `server-side-rendering.mdx` & `v2.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updated `from-gatsby.mdx`,`from-nuxtjsmdx`, `from-nextjs.mdx`, `server-side-rendering.mdx` & `v2.mdx` based on this commit 0b7b49471a1b0fdcb64a667a0a87b2de8617f4fe

### CODE REVIEW

File: src/content/docs/es/guides/server-side-rendering.mdx [hunk 1]
```diff
@@ -124,38 +155,23 @@ layout: '../layouts/markdown.astro'
...
-  output: 'hybrid',
-  experimental: {
-    hybridOutput: true,
-  },
-});
-```
+### Optar por no pre-renderizar
 
-Con esta configuración, todas tus páginas y endpoints de API se pre-renderizarán, incluso si tu proyecto está configurado con un adaptador para usar SSR. Luego puedes *optar por no hacerlo* de la representación previa para cualquier archivo o ruta individual agregando `export const prerender = false` a los archivos que deben ser renderizados en el servidor.
+Para la mayoría de los sitios estáticos configurados como `output: hybrid`, añade `export const prerender = false` a cualquier archivo que debería ser renderizado en el servidor:
```
Comment (Waxer59) [inline src/content/docs/es/guides/server-side-rendering.mdx:174]:
```suggestion
Para un sitio principalmente estático configurado como `output: hybrid`, añade `export const prerender = false` a cualquier archivo que debería ser renderizado en el servidor:
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Review by Waxer59 — APPROVED** [review]
LGTM ✔️

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks folks 🥳

**Pull request merged**